test/test_helper.rb

```diff
@@ -5,12 +5,13 @@
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 
-require 'adaptive_alias'
 require 'minitest/autorun'
 
 require 'lib/mysql2_connection'
 
 require 'acts-as-taggable-on'
+
+require 'adaptive_alias'
```

Comment:
move this line to below `require 'lib/mysql2_connection'` in order to let `ActiveModel::AttributeMethods` be included first and then include adaptive_alias patch, which will cause problem in ruby 2.6, 2.7.